src/services/ui/src/features/package-actions/lib/index.ts

```diff
@@ -53,8 +59,19 @@ export const submitActionForm = async ({
     });
     alert.setBannerStyle("success");
     alert.setBannerShow(true);
-    alert.setBannerDisplayOn(path);
-    navigate({ path });
+    alert.setBannerDisplayOn(
+      locationState.from?.includes("dashboard") ? "/dashboard" : "/details",
```

Comment:
we're backing ourselves into a binary corner of sorts with this adjustment, and i'd prefer we handling setting a `path` to display on and navigate to to avoid the limitations introduced.

Reply:
That's fair. Can I show the issues I was having when you have some free time. Maybe it could have been a bug

Reply:
Let's sync Monday, because there's a similar issue called out in an upcoming ticket! we might 2-birds this thing.